pyproject.toml

```diff
@@ -11,6 +11,7 @@ python = "^3.12"
 httpx = "^0.27.2"
 asyncstdlib = "^3.13.0"
 adcm-version = "^1.0.3"
+pyyaml = "^6.0.2"
```

Comment:
I think this dependency can be placed in the `test` group. @Starovoitov @Sealwing What do you think?

Reply:
moved it under test section